---
title: Integration of single-nuclei RNA-sequencing, spatial transcriptomics and histochemistry
  defines the complex microenvironment of NF1-associated plexiform neurofibromas

# Authors
# A YAML list of author names
# If you created a profile for a user (e.g. the default `admin` user at `content/authors/admin/`), 
# write the username (folder name) here, and it will be replaced with their full name and linked to their profile.
authors:
- Vladimir Amani
- Kent A Riemondy
- Rui Fu
- Andrea M Griesinger
- Enrique Grimaldo
- Graziella Ribeiro De Sousa
- Ahmed Gilani
- Molly Hemenway
- Nicholas K Foreman
- Andrew M Donson
- Nicholas Willard

# Author notes (such as 'Equal Contribution')
# A YAML list of notes for each author in the above `authors` list
author_notes: []

date: '2023-09-01'

# Date to publish webpage (NOT necessarily Bibtex publication's date).
publishDate: '2023-11-03T18:16:10.861262Z'

# Publication type.
# A single CSL publication type but formatted as a YAML list (for Hugo requirements).
publication_types:
- article-journal

# Publication name and optional abbreviated publication name.
publication: '*Acta Neuropathol Commun*'
publication_short: ''

doi: 10.1186/s40478-023-01639-1

abstract: Plexiform neurofibroma (PN) is a leading cause of morbidity in children
  with the genetic condition Neurofibromatosis Type 1 (NF1), often disfiguring or
  threatening vital structures. During formation of PN, a complex tumor microenvironment
  (TME) develops, with recruitment of neoplastic and non-neoplastic cell types being
  critical for growth and progression. Due to the cohesive cellularity of PN, single-cell
  RNA-sequencing is difficult and may result in a loss of detection of critical cellular
  subpopulations. To bypass this barrier, we performed single-nuclei RNA-sequencing
  (snRNA-seq) on 8 frozen PN samples, and integrated this with spatial transcriptomics
  (ST) in 4 PN samples and immunohistochemistry to provide morphological context to
  transcriptomic data. SnRNA-seq analysis definitively charted the heterogeneous cellular
  subpopulations in the PN TME, with the predominant fraction being fibroblast subtypes.
  PN showed a remarkable amount of inter-sample homogeneity regarding cellular subpopulation
  proportions despite being resected from a variety of anatomical locations. ST analysis
  identified distinct cellular subpopulations which were annotated using snRNA-seq
  data and correlated with histological features. Schwann cell/fibroblast interactions
  were identified by receptor/ligand interaction analysis demonstrating a high probability
  of Neurexin 1/Neuroligin 1 (NRXN1/NLGN1) receptor-ligand cross-talk predicted between
  fibroblasts and non-myelinated Schwann cells (NM-SC) and subtypes, respectively.
  We observed aberrant expression of NRXN1 and NLGN1 in our PN snRNA-seq data compared
  to a normal mouse sciatic nerve single-cell RNA-seq dataset. This pathway has never
  been described in PN and may indicate a clear and direct communication pathway between
  putative NM-SC cells of origin and surrounding fibroblasts, potentially driving
  disease progression. SnRNA-seq integrated with spatial transcriptomics advances
  our understanding of the complex cellular heterogeneity of PN TME and identify potential
  novel communication pathways that may drive disease progression, a finding that
  could provide translational therapy options for patients with these devastating
  tumors of childhood and early adulthood.

# Summary. An optional shortened abstract.
summary: ''

tags: []

# Display this page in a list of Featured pages?
featured: false

# Links
url_pdf: ''
url_code: ''
url_dataset: ''
url_poster: ''
url_project: ''
url_slides: ''
url_source: ''
url_video: ''

# Custom links (uncomment lines below)
# links:
# - name: Custom Link
#   url: http://example.org

# Publication image
# Add an image named `featured.jpg/png` to your page's folder then add a caption below.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects: ['internal-project']` links to `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects: []
links:
- name: URL
  url: http://dx.doi.org/10.1186/s40478-023-01639-1
---

Add the **full text** or **supplementary notes** for the publication here using Markdown formatting.
