---
title: Spatial transcriptomic analysis delineates epithelial and mesenchymal subpopulations
  and transition stages in childhood ependymoma

# Authors
# A YAML list of author names
# If you created a profile for a user (e.g. the default `admin` user at `content/authors/admin/`), 
# write the username (folder name) here, and it will be replaced with their full name and linked to their profile.
authors:
- Rui Fu
- Gregory A Norris
- Nicholas Willard
- Andrea M Griesinger
- Kent A Riemondy
- Vladimir Amani
- Enrique Grimaldo
- Faith Harris
- Todd C Hankinson
- Siddhartha Mitra
- Timothy A Ritzmann
- Richard R Grundy
- Nicholas K Foreman
- Andrew M Donson

# Author notes (such as 'Equal Contribution')
# A YAML list of notes for each author in the above `authors` list
author_notes: []

date: '2022-10-01'

# Date to publish webpage (NOT necessarily Bibtex publication's date).
publishDate: '2023-11-03T18:16:10.846163Z'

# Publication type.
# A single CSL publication type but formatted as a YAML list (for Hugo requirements).
publication_types:
- article-journal

# Publication name and optional abbreviated publication name.
publication: '*Neuro. Oncol.*'
publication_short: ''

doi: 10.1093/neuonc/noac219

abstract: 'BACKGROUND: The diverse cellular constituents of childhood brain tumor
  ependymoma, recently revealed by single cell RNA-sequencing, may underly therapeutic
  resistance. Here we use spatial transcriptomics to further advance our understanding
  of the tumor microenvironment, mapping cellular subpopulations to the tumor architecture
  of ependymoma posterior fossa subgroup A (PFA), the commonest and most deadly childhood
  ependymoma variant. METHODS: Spatial transcriptomics data from intact PFA sections
  was deconvoluted to resolve the histological arrangement of neoplastic and non-neoplastic
  cell types. Key findings were validated using immunohistochemistry, in vitro functional
  assays and outcome analysis in clinically-annotated PFA bulk transcriptomic data.
  RESULTS: PFA are comprised of epithelial and mesenchymal histological zones containing
  a diversity of cellular states, each zone including co-existing and spatially distinct
  undifferentiated progenitor-like cells; a quiescent mesenchymal zone population,
  and a second highly mitotic progenitor population that is restricted to hypercellular
  epithelial zones and that is more abundant in progressive tumors. We show that myeloid
  cell interaction is the leading cause of mesenchymal transition in PFA, occurring
  in zones spatially distinct from hypoxia-induced mesenchymal transition, and these
  distinct EMT-initiating processes were replicated using in vitro models of PFA.
  CONCLUSIONS: These insights demonstrate the utility of spatial transcriptomics to
  advance our understanding of ependymoma biology, revealing a clearer picture of
  the cellular constituents of PFA, their interactions and influence on tumor progression.'

# Summary. An optional shortened abstract.
summary: ''

tags:
- ependymoma; spatial transcriptomics; tumor microenvironment

# Display this page in a list of Featured pages?
featured: false

# Links
url_pdf: ''
url_code: ''
url_dataset: ''
url_poster: ''
url_project: ''
url_slides: ''
url_source: ''
url_video: ''

# Custom links (uncomment lines below)
# links:
# - name: Custom Link
#   url: http://example.org

# Publication image
# Add an image named `featured.jpg/png` to your page's folder then add a caption below.
image:
  caption: ''
  focal_point: ''
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects: ['internal-project']` links to `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects: []
links:
- name: URL
  url: http://dx.doi.org/10.1093/neuonc/noac219
---

Add the **full text** or **supplementary notes** for the publication here using Markdown formatting.
